pyzx/genetic.py

```diff
@@ -0,0 +1,156 @@
+from tqdm import tqdm
```

Comment:
Missing the standard header on top of the file. Also should have a top level documentation comment. Also, there are no documentation strings for all the classes.